src/blocks/mrc_steps.ts

```diff
@@ -75,6 +76,9 @@ const STEPS = {
   loadExtraState: function (this: StepsBlock, state: StepsExtraState): void {
     this.mrcStepNames = state.stepNames;
     this.updateShape_();
+    
+    // Update all mrc_jump_to_step blocks to recheck validity
+    this.mrcCheckJumpBlocks();
```

Comment:
I don't think it's appropriate to check other blocks during this block's loadExtraState, since other blocks might not have been created yet (right?).

Reply:
You are correct.  I removed it.